packages/artifact/README.md

```diff
@@ -106,6 +107,19 @@ const {downloadPath} = await artifact.downloadArtifact(id, {
 console.log(`Downloaded artifact ${id} to: ${downloadPath}`)
 ```
 
+### Delete an Artifact
+
+To delete an artifact, all you need is the name. Also supports options to delete from other repos/runs given a token with proper permissions is supplied.
```

Comment:
Could we add a separate example here for deleting an artifact from another workflow run with the `findBy` options? Just so it's super obvious for anyone going through the documentation

Permissions might be a bit vague here. `GITHUB_TOKEN` won't work but you rather need a PAT with the repository write permission I think

Reply:
Yea that's a good idea

Reply:
Added in https://github.com/actions/toolkit/pull/1626/commits/1852eb2115645114d47975ca9eef48d4e56b17b0